src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  flex-wrap: wrap;
```

Comment:
flex-direction and flex-wrap are redundant here
all card elements started from new line by default